src/components/Books.js

```diff
@@ -13,16 +13,28 @@ const Books = () => {
   }, []);
   return (
     <div>
-      List of Books will be displayed here:
-
       {
-            book.map((element) => (
+      book.map((element) => (
```

Comment:
- [ ] Nice working on your design, kindly adjust the book section to look exactly like what was given in the template. see the screenshot below:

### Your Design
![your-img](https://user-images.githubusercontent.com/48371156/202838812-4150f54c-48cb-43f5-a20d-1c5f8c220f7d.png)

### Template

![tep-img](https://user-images.githubusercontent.com/48371156/202838844-b2ac65d9-6c0c-47b4-819a-e5874ef59bda.png)